use_case_examples/hybrid_model/compile.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+VIA_MLIR=0
```

Comment:
why using a bash script file + `os.environ.get("INDEX", 2)` in the compile.py than use regular argparse ? 😅 I think we should keep it simple

Reply:
I find env vars easier to use than argparse personnaly.